custom_components/daikin_onecta/sensor.py

```diff
@@ -46,51 +46,28 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
         "climateControlMainZone",
     }
     for dev_id, device in hass.data[DAIKIN_DOMAIN][DAIKIN_DEVICES].items():
-        managementPoints = device.daikin_data.get("managementPoints", [])
-        for management_point in managementPoints:
+        management_points = device.daikin_data.get("managementPoints", [])
+        for management_point in management_points:
             management_point_type = management_point["managementPointType"]
```

Comment:
We have a lot of diagnostics sensors which are lost because of this check as far as I can tell without a test

Reply:
I was thinking we maybe could add a flag to the config flow process to check if the user wants to add the diagnostics. As far as I'm concerned I don't need that on HA.

Reply:
You can disable them in HA when you don't need them. As most of them are diagnostics they will not be added by default to a dashboard